Octokit/Models/Response/RepositoryInvitation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum InvitationPermissionType
```

Comment:
There is already a `RepositoryPermissions` enum that should be used for these values